apps/builder/app/builder/features/sidebar-left/sidebar-left.tsx

```diff
@@ -62,17 +64,37 @@ export const SidebarLeft = ({ publish }: SidebarLeftProps) => {
           ))}
         </SidebarTabsList>
         <Box css={{ borderRight: `1px solid  ${theme.colors.borderMain}` }}>
-          <SidebarTabsTrigger
-            as={"button"}
-            aria-label={"Report bug"}
-            onClick={() => {
-              window.open(
-                "https://github.com/webstudio-is/webstudio/discussions/new?category=q-a&labels=bug&title=[Bug]"
-              );
-            }}
+          <HelpPopover>
+            <Tooltip
+              side="top"
+              content="Ask for help on Discord"
```

Comment:
its not just discord any more, we need a better tooltip text here, @taylornowotny - its the tooltip that is shown when hovering the help icon

Reply:
"Learn Webstudio or ask for help" is the right text